scripts/common.py

```diff
@@ -223,8 +223,8 @@
     'logged-out-user/click-all-buttons-on-navbar',
     'logged-out-user/click-all-links-in-about-oppia-footer',
     'logged-out-user/click-all-links-on-get-started-page',
-    'logged-in-user/subscribe-to-creator-and-view-all-explorations-' +
-        'by-that-creator',
+    'logged-in-user/subscribe-to-creator-and-view-all-explorations-'
+    'by-that-creator',
```

Comment:
I think this is actually harder to read without the indentation. When quickly scanning this list, most readers will assume that each line is a list entry, but that's not the case for `by-that-creator`. I recommend either keeping the indentation or indenting the whole list entry and wrapping in parentheses like this:

```python
'logged-out-user/click-all-links-on-get-started-page',
(
    'logged-in-user/subscribe-to-creator-and-view-all-explorations-'
    'by-that-creator'
),
```